src/app/view/components/dot-navigation/dot-navigation.service.ts

```diff
@@ -38,17 +38,12 @@ export class DotNavigationService {
             this.reloadNavigation();
         });
 
-        this.loginService.auth$.filter((auth: Auth) => !!(auth.loginAsUser || auth.user)).subscribe((auth: Auth) => {
-            this.reloadNavigation().subscribe((isPortletInMenu: boolean) => {
-                const isUserRedirect = auth.user['editModeUrl'] || this.dotRouterService.previousSavedURL;
-
-                if (isUserRedirect) {
-                    this.userCustomRedirect(auth.user['editModeUrl']);
-                } else if (!isPortletInMenu) {
-                    this.goToFirstPortlet();
-                }
+        this.loginService.auth$
```

Comment:
Now we don't have to worry about doing the redirect after login here, it never make sense to me doing that here.